influunt-app/app/views/controladores/wizard/aneis.html

```diff
@@ -11,148 +11,127 @@
-            </ul>
-            <div class="ui-layout-content tab-content">
-              <div id="tabs-1">
-                <div class="col-lg-12 col-sm-12 text-left">
-                  <h2>{{ 'controladores.anel' | translate }} {{ currentAnel.posicao }}</h2>
-                </div>
-                <div class="col-lg-9 col-sm-9 groups ibox-content">
-                  <div class="col-lg-6 col-sm-6 no-padding-left">
-                    <div class="col-lg-6 col-sm-6">
-                      <div class="form-group">
-                        <label class="control-label">{{ 'controladores.gruposPedestres' | translate }}</label>
-                        <input type="number" name="quantidadeGrupoPedestre" class="form-control" ng-model="currentAnel.quantidadeGrupoPedestre" />
-                      </div>
-                    </div>
-                    <div class="col-lg-6 col-sm-6">
-                      <div class="form-group">
-                        <label class="control-label">{{ 'controladores.gruposSinaisVeiculares' | translate }}</label>
-                        <input type="number" name="quantidadeGrupoVeicular" placeholder="" class="form-control" data-ng-model="currentAnel.quantidadeGrupoVeicular">
-                      </div>
-                    </div>
-                    <div class="col-lg-6 col-sm-6">
-                      <div class="form-group">
-                        <label class="control-label">{{ 'controladores.numeroDetectores' | translate }}</label>
-                        <input type="number" name="quantidadeDetectorVeicular" placeholder="" class="form-control" data-ng-model="currentAnel.quantidadeDetectorVeicular">
-                      </div>
+        <div class="col-lg-12 col-sm-12 padding-top-20 tabs-container">
+          <influunt-tabs class="tabpanel"
+                         data-on-add="ativarProximoAnel"
+                         data-on-remove="desativarUltimoAnel"
+                         data-on-activate="selecionarCurrentAnel"
+                         data-aneis-ativos="aneisAtivos"
+                         data-error-check="anelTemErro"
+                         data-max-tabs="objeto.limiteAnel"></influunt-tabs>
+          <div class="ui-layout-content tab-content">
+            <div id="tabs-1">
+              <div class="col-lg-12 col-sm-12 text-left">
+                <h2>{{ 'controladores.anel' | translate }} {{ currentAnel.posicao }}</h2>
+              </div>
+              <div class="col-lg-9 col-sm-9 groups ibox-content">
+                <div class="col-lg-6 col-sm-6 no-padding-left">
+                  <div class="col-lg-6 col-sm-6">
+                    <div class="form-group">
+                      <label class="control-label">{{ 'controladores.gruposPedestres' | translate }}</label>
+                      <input type="number" name="quantidadeGrupoPedestre" class="form-control" ng-model="currentAnel.quantidadeGrupoPedestre" />
```

Comment:
Esse campo foi removido da API, também deve ser feito na APP. Se não for fazer nesse PR criar uma Issue